### PR TITLE
Fix issues for case check_dpdk_build.

### PR DESCRIPTION
**Pull request by LiliDeng**
After fix.
```
2021-09-05 05:24:22.599[19548][INFO] lisa.RootRunner ________________________________________
2021-09-05 05:24:22.600[19548][INFO] lisa.RootRunner                              Dpdk.check_dpdk_build: PASSED
```
Contains fix for #1505

### CODE REVIEW

File: microsoft/testsuites/dpdk/dpdksuite.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ class Dpdk(TestSuite):
             This test case checks DPDK can be built and installed correctly.
         """,
         requirement=simple_requirement(
+            min_nic_count=2,
```
Comment (squirrelsc) [inline microsoft/testsuites/dpdk/dpdksuite.py:34]: Why does it need at least two nic?

File: microsoft/testsuites/dpdk/dpdktestpmd.py [hunk 2]
```diff
@@ -179,7 +183,7 @@ def _install_dependencies(self) -> bool:
 
     def _generate_testpmd_command(self, nic_to_include: str) -> str:
         return (
-            f"{self._testpmd_install_path} -l 2,3 -n 4 --proc-type=primary "
+            f"{self._testpmd_install_path} -l 0-1 -n 4 --proc-type=primary "
```
Comment (squirrelsc) [inline microsoft/testsuites/dpdk/dpdktestpmd.py:196]: Add comments to explain the magic numbers.

File: microsoft/testsuites/dpdk/dpdksuite.py (hunk 1 above)
Comment (LiliDeng) [reply microsoft/testsuites/dpdk/dpdksuite.py:34]: https://docs.microsoft.com/en-us/azure/virtual-network/setup-dpdk#prerequisites
image

Comment (squirrelsc) [reply microsoft/testsuites/dpdk/dpdksuite.py:34]: add comments to explain it. The preconditions should be a part of the description.

Comment (LiliDeng) [reply microsoft/testsuites/dpdk/dpdksuite.py:34]: fixed

File: microsoft/testsuites/dpdk/dpdktestpmd.py (hunk 2 above)
Comment (LiliDeng) [reply microsoft/testsuites/dpdk/dpdktestpmd.py:196]: added

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**